content/en/docs/guides/custom-load-balancing.md

```diff
@@ -5,6 +5,8 @@ description: >-
   under unique circumstances.
 ---
 
+{{< youtube 7LYvl-nr0t8 youtube-video >}}
```

Comment:
Please use the title param to mention what the video is about. It will also help the readers of the code know the contents without opening them.

See https://gohugo.io/content-management/shortcodes/#youtube

Reply:
This can be marked as resolved.